information.html

```diff
@@ -118,6 +139,12 @@
                 the ferry terminal is on the north-west of the city.
             </p>
 
+            <h4>By Hovercraft</h4>
+            <p>
+                Arrive in style from the Isle of Wight; check out <a href="http://www.hovertravel.co.uk/">the Hover
+                Travel</a> website for information about a speedy trip from IoW.
```

Comment:
I was confused because I'm on the IoW right now, and thought the conference was here for a moment